packages/hooks/src/usePaymentAgentTransferVisible.ts

```diff
@@ -1,11 +1,7 @@
 import { useFetch } from '@deriv/api';
-import { useStore } from '@deriv/stores';
 
 const usePaymentAgentTransferVisible = () => {
-    const { client } = useStore();
-    const { is_authorize } = client;
-
-    const { data, ...rest } = useFetch('get_settings', { options: { enabled: is_authorize } });
```

Comment:
Remove `options-enabled-is_authorize` because authorize check is now happening inside useFetch hook.